src/Symfony/Component/Messenger/Command/FailedMessagesShowCommand.php

```diff
@@ -38,6 +38,8 @@ protected function configure(): void
             ->setDefinition([
                 new InputArgument('id', InputArgument::OPTIONAL, 'Specific message id to show'),
                 new InputOption('max', null, InputOption::VALUE_REQUIRED, 'Maximum number of messages to list', 50),
+                new InputOption('group', null, InputOption::VALUE_NONE, 'Group by message class'),
```

Comment:
When looking at implementation, the wording is wrong. I expect the same behavior with messages grouped by class, but this looks like statistic per classes.

Maybe `stats` would be better?